bindings_test.go

```diff
@@ -983,6 +984,7 @@ func TestFunctionParameters(t *testing.T) {
 	}
 
 	runDBTest(t, func(dbt *DBTest) {
+		dbt.exec("ALTER SESSION SET BIND_NULL_VALUE_USE_NULL_DATATYPE=false")
```

Comment:
It may be better to use `mustExec` and ensure that no error is returned.

Reply:
done